packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -134,7 +137,13 @@ export default {
         labelStyle: {
             type: String,
             default: 'default',
-            validator: value => (VALID_LABEL_STYLES.indexOf(value) !== -1) // The prop value must match one of the valid input types
+            validator: value => (VALID_LABEL_STYLES.indexOf(value) !== -1) // The prop value must match one of the valid label types
+        },
+
+        fieldSize: {
```

Comment:
Would be good to add some tests for this (can be a separate PR though).

Reply:
I will add for this PR, thanks!